packages/devtools_app/lib/src/screens/inspector_v2/widget_properties/properties_view.dart

```diff
@@ -0,0 +1,138 @@
+// Copyright 2024 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:devtools_app_shared/ui.dart';
+import 'package:flutter/material.dart';
+
+import '../../../shared/diagnostics/diagnostics_node.dart';
+import '../inspector_controller.dart';
+
+class PropertiesView extends StatelessWidget {
+  const PropertiesView({
+    super.key,
+    required this.controller,
+    required this.node,
+  });
+
+  final InspectorController controller;
+  final RemoteDiagnosticsNode node;
+
+  @override
+  Widget build(BuildContext context) {
+    return ValueListenableBuilder<List<RemoteDiagnosticsNode>>(
+      valueListenable: controller.selectedNodeProperties,
+      builder: (context, properties, _) {
+        return Container(
+          margin: const EdgeInsets.all(denseSpacing),
+          child: ListView.builder(
+            itemCount: properties.length,
+            itemBuilder: (context, index) {
+              return PropertyItem(
+                index: index,
+                properties: properties,
+              );
+            },
+          ),
+        );
+      },
+    );
+  }
+}
+
+class PropertyItem extends StatelessWidget {
+  const PropertyItem({
+    super.key,
+    required this.properties,
+    required this.index,
+  });
+
+  final List<RemoteDiagnosticsNode> properties;
+  final int index;
+
+  @override
+  Widget build(BuildContext context) {
+    final property = properties[index];
+    final theme = Theme.of(context);
+
+    return DecoratedBox(
+      decoration: BoxDecoration(
+        border: _calculateBorderForRow(
+          rowIndex: index,
+          theme: theme,
+        ),
+        color: alternatingColorForIndex(
+          index,
+          Theme.of(context).colorScheme,
```

Comment:
use the local var `theme` instead of looking this up again. Also super nit I think this method call and the one above will now fit on a single line

Reply:
Done!